pom.xml

```diff
@@ -83,7 +83,7 @@
 
   <modules>
     <module>planetiler-core</module>
-    <module>planetiler-basemap</module>
+    <module>planetiler-openmaptiles</module>
```

Comment:
I assume you'll want to build/run the planetiler-openmaptiles on its own from the standalone repo? If so we'll want to think through how we are going to structure the pom.xml's so it can be run both standalone and as a child project here.

One option is what the [planetiler-examples](https://github.com/onthegomap/planetiler/tree/main/planetiler-examples) submodule does - it has 2 pom.xml's, one for standalone and one for a child module (although we could rename so main pom.xml is for standalone). We'll need to keep the 2 in sync but shouldn't be too bad unless you start adding a lot of dependencies.

Reply:
OK, sounds good for now - I could help set that up later.